### PR TITLE
[RCP][unet3d] Replace BS=32 RCPs from AMP with FP32

### PR DESCRIPTION
**Pull request by shangw-nvidia**
@emizan76 @xyhuang We are updating the unet3d BS=32 RCP. The previously proposed RCPs were from AMP runs. This PR contains the its FP32 counterparts.

### CODE REVIEW

**Review by emizan76 — COMMENTED** [review]
Can you add the rest of the hyperparameters, learning rate and warmup epochs are the only ones used for Unet3D? This applies also to the other unet3d RCP.

**Comment by shangw-nvidia** [comment]
> Can you add the rest of the hyperparameters, learning rate and warmup epochs are the only ones used for Unet3D? This applies also to the other unet3d RCP.

I just pushed a commit that adds in the rest of the hyper-parameters.

Note that, the training policy on unet3d's hyper-parameters seems to be out-of-date: https://github.com/mlcommons/training_policies/blob/master/training_rules.adoc#91-hyperparameters . For example:
- `opt_learning_rate_warmup_steps` -> `opt_learning_rate_warmup_epochs`
- `opt_learning_rate_decay_steps` -> `opt_learning_rate_decay_boundary_epochs`

Going forward, I would suggest to separate the tags in https://github.com/mlcommons/logging/blob/master/mlperf_logging/mllog/constants.py into one file per benchmark; otherwise it's fairly cumbersome to figure out which tags belongs to which benchmarks without looking into the reference code in detail.

**Review by emizan76 — APPROVED** [review]
